docs-ref-conceptual/install-azure-cli-yum.md

```diff
@@ -57,14 +57,42 @@ Here are some common problems seen when installing with `yum`. If you experience
 
 ### Install on RHEL 7.6 or other systems without Python 3
 
-If you can, please upgrade your system to a version with official support for `python3` package. Otherwise, you need to first install a `python3` package, either [build from source](https://github.com/linux-on-ibm-z/docs/wiki/Building-Python-3.6.x) or install through some [additional repo](https://developers.redhat.com/blog/2018/08/13/install-python3-rhel/). Then you can download the package and install it without dependency.
+If you can, please upgrade your system to a version with official support for `python3` package. Otherwise, you need to first install a `python3` package, you can choose to build it from source. Azure CLI requires `SSL 1.1+` and you need to build `openssl 1.1` from source first before building `python3`:
```

Comment:
I'm curious about why there is no such information before.

Reply:
Actually, I tried with `openssl 1.0.2` and the install can also succeed. But `openssl 1.0.2` has reached `end-of-life` at the end of 2019, so we should use `1.1+`.

With newer Linux systems, `openssl` should already be installed.